chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/TestFunctionalityDTO.java

```diff
@@ -77,17 +74,16 @@ public void setPracticeType(final PracticeTypeDTO practiceType) {
         this.practiceType = practiceType;
     }
 
-    /**
-     * @return the certificationCriterion
-     */
-    public CertificationCriterionDTO getCertificationCriterion() {
-        return certificationCriterion;
-    }
-
-    /**
-     * @param certificationCriterion the certificationCriterion to set
-     */
-    public void setCertificationCriterion(final CertificationCriterionDTO certificationCriterion) {
-        this.certificationCriterion = certificationCriterion;
+    @Override
+    public boolean equals(Object o) {
```

Comment:
I may be dating myself here, but last time I looked if you override equals then you also were supposed to override hashcode. Is this still a thing?

Reply:
Done